### ISSUE TITLE
DVS Gesture Dataset Size Doesn't Match

### ISSUE THREAD

**Issue by Po0ria** [fangwei123456/Parametric-Leaky-Integrate-and-Fire-Spiking-Neuron#8]
This is the script that I am using:
`python ./codes/train_val.py -init_tau 2.0 -use_max_pool -device cuda:0 -dataset_name DVS128Gesture -log_dir_prefix ./pretrained_train_val -T 20 -max_epoch 1024  -detach_reset -channels 128 -number_layer 5 -split_by number -normalization None -use_plif`

Here is what you guys logged in ./logs/A/dvsgesture.txt : 
`train dataset len 803, val dataset len 133`
Here is what I get from the dataset downloaded from official IBM website:
`train dataset len 923, val dataset len 157`

**Comment by fangwei123456** [comment]
Note that `dvsgesture.txt` is located in `B` rather than `A`.

https://github.com/fangwei123456/Parametric-Leaky-Integrate-and-Fire-Spiking-Neuron/blob/f644ff652f8551021c8b45d5702f38b92a3bce72/logs/B/terminal_out/dvsgesture.txt#L10

Acc B is the accuracy with a validation set.
![image](https://user-images.githubusercontent.com/38352109/188530520-21a2c620-0bb1-4f52-9690-886578e806ad.png)

**Comment by Po0ria** [comment]
Regarding:  

>  Note that dvsgesture.txt is located in B rather than A. 

You're completely right. But from my understanding, DVS Gesture has 1080 training sample. according to the paper 15% goes to validation and 85% goes to the new training set:
length of the new training set = 923
length of the new validation set = 157
I believe the size of my dataset is different than the one that you are using for some reason. Because assuming the total size of 1342 instances (from the DVS gesture paper (Amir et al. 2017) and split data in the default ratio of 80:20, it makes sense to have 1080 data for training.

**Comment by fangwei123456** [comment]
I think the origin dataset I used at that time was not complete because I only got `train dataset len 803, val dataset len 133`. The correct result should be `train dataset len 1000, val dataset len 176`. In general, a smaller training set will cause a lower test accuracy.


> Because assuming the total size of 1342 instances 

Thanks for your notification. I check the samples number and find some difference. Now I have added the information in API doc:

https://spikingjelly.readthedocs.io/zh_CN/latest/spikingjelly.datasets.html#spikingjelly.datasets.dvs128_gesture.DVS128Gesture

![image](https://user-images.githubusercontent.com/38352109/188805319-72074704-a383-42e4-8603-732e40f5dcbb.png)
